lib/config/definitions.js

```diff
@@ -1369,6 +1369,21 @@ const options = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'gradle-wrapper',
+    description: 'Configuration object for gradle-wrapper.properties files',
+    stage: 'package',
+    type: 'json',
+    default: {
+      enabled: true,
+      fileMatch: ['(^|/)gradle/wrapper/gradle-wrapper.properties$'],
+      typeStrategy: 'auto',
+      digestsStrategy: 'true',
+      serviceUrl: 'https://services.gradle.org/versions/all',
```

Comment:
Is there a use case where someone would need to change typeStrategy, digestsStrategy, or serviceUrl?

Reply:
It was initially more a 'why not' then a use case. I'm not a big fan of hardcoded urls, specially if there already is a good configuration system. More power to the user.

* Replacing the used gradle build or supported version within a company

Or is this a security problem for the renovate service?

Reply:
The `repositoryUrls` field could be used if you want to allow the user to override the default URL.

My main concern for the other two is polluting the configuration space with options that may not be used but must be "maintained" forever. I prefer to keep the same type/digests strategy as exists in the file for that reason.